app/assess/routes.py

```diff
@@ -30,12 +34,71 @@ def funds():
     from fund store
     :return:
     """
-
     funds = get_funds()
-
     return render_template("funds.html", funds=funds)
 
 
+@login_requested
+@assess_bp.route(
+    "score/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>",
+    methods=["POST", "GET"],
+)
+def application_sub_crit_scoring(application_id: str, sub_criteria_id: str):
+    fund = get_fund(Config.COF_FUND_ID)
+    form = JustScoreForm()
```

Comment:
If we renamed `JustScore` - it might be good to also rename this form, minor though